app/views/portal/show.rb

```diff
@@ -0,0 +1,871 @@
+            },
+            {
+              title: 'site.object.meta-label.version',
+              fields: ['proxies.dctermsHasVersion']
+            },
+            {
+              title: 'site.object.meta-label.is-format-of',
+              fields: ['proxies.dctermsIsFormatOf']
+            },
+            {
+              title: 'site.object.meta-label.is-referenced-by',
+              fields: ['proxies.dctermsIsReferencedBy']
+            },
+            {
+              title: 'site.object.meta-label.is-replaced-by',
+              fields: ['proxies.dctermsIsReplacedBy']
+            },
+            {
+              title: 'site.object.meta-label.is-required-by',
+              fields: ['proxies.dctermsIsRequiredBy']
+            },
+            {
+              title: 'site.object.meta-label.edm.has-met',
+              fields: ['proxies.edmHasMet']
+            },
+            {
+              title: 'site.object.meta-label.edm.incorporates',
+              fields: ['proxies.edmIncorporates']
+            },
+            {
+              title: 'site.object.meta-label.edm.is-derivative-of',
+              fields: ['proxies.edmIsDerivativeOf']
+            },
+            {
+              title: 'site.object.meta-label.edm.is-representation-of',
+              fields: ['proxies.edmIsRepresentationOf']
+            },
+            {
+              title: 'site.object.meta-label.edm.is-similar-to',
+              fields: ['proxies.edmIsSimilarTo']
+            },
+            {
+              title: 'site.object.meta-label.edm.is-successor-of',
+              fields: ['proxies.edmIsSuccessorOf']
+            },
+            {
+              title: 'site.object.meta-label.edm.realises',
+              fields: ['proxies.edmRealizes']
+            },
+            {
+              title: 'site.object.meta-label.edm.was-present-at',
+              fields: ['proxies.edmRealizes']
+            }
+          ]
+        ),
+        similar: {
+          title: t('site.object.similar-items') + ':',
+          more_items_query: search_path(mlt: document.id),
+          items: @similar.map do |doc|
+            {
+              url: document_path(doc, format: 'html'),
+              title: render_document_show_field_value(doc, ['dcTitleLangAware', 'title']),
+              img: {
+                alt: render_document_show_field_value(doc, ['dcTitleLangAware', 'title']),
+                src: render_document_show_field_value(doc, 'edmPreview')
+              }
+            }
+          end
+        },
+        thumbnail: render_document_show_field_value(document, 'europeanaAggregation.edmPreview', tag: false)
+      }
+    end
+
+    def labels
+      {
+        show_more_meta: t('site.object.actions.show-more-data'),
+        show_less_meta: t('site.object.actions.show-less-data'),
+        rights: t('site.object.meta-label.rights-human')
+      }
+    end
+
+    private
+
+    def edm_is_shown_by_download_url
```

Comment:
Assignment Branch Condition size for edm_is_shown_by_download_url is too high. [15.43/15]